Source/Core/BaseRow.swift

```diff
@@ -127,6 +127,13 @@ open class BaseRow: BaseRowType {
 
     open func prepare(for segue: UIStoryboardSegue) {}
 
+    /**
+     Helps to pick destination part of the cell after scrolling
+     */
+    open var isScrollToTop: Bool {
```

Comment:
This is a compute property so that to change it you must subclass the Row. I think it would be better as a simple public variable.
Apart from that, as you open the possibility to change where the row should be scrolled to why not let the variable be a `UITableViewScrollPosition`?

I would say something like:
`public var inlineRowScrollPosition: UITableViewScrollPosition = .bottom`

And for one line documentation you should use `///` if possible to shorten it a bit.

What do you think?